src/middlewares/auth/utils/isAuthHandlerByUserIdFromBody.js

```
@@ -0,0 +1,13 @@
+const UnauthorizedError = require("../../../exceptions/UnauthorizedError");
+
+const isAuthHandlerByUserIdFromBody = async (req, res) => {
+  const userId = await req.body.userId;
+
+  if (!userId) return false;
```

IsAuthHandler should throw Error (instead of returning false).